feature/experimental/system/gnmi/benchmarking/ate_tests/full_configuration_replace_test/full_configuration_replace_test.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package full_configuration_replace_test
+
+import (
+	"sort"
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup"
+	"github.com/openconfig/featureprofiles/internal/args"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygot/ygot"
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+// Topology:
+// 	dut:port(1..N)
```

Comment:
This two-line comment is a little unclear about what it is communicating. That there is some N number of ports? I suspect it's duplicative with whatever is in the README (or should be in the README) and can be safely dropped here.

Reply:
I have added a note for user convenience. Please see if it's clear now.

Reply:
I don't think the comments really add much beyond what is already understood about the nature of these tests. I'd just drop.

Reply:
Removed the comment.